### PR TITLE
Initial - 0.0.1

### PR DESCRIPTION
**Pull request by usirin**

### CODE REVIEW

File: src/compiler.coffee [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    { children } = node
+
+    compiled = []
+
+    compiled.push "new KDViewNode({"
+    compiled.push "tagName: '#{tagName}',"  if tagName?
+    compiled.push "domId: '#{domId}',"  if domId?
+    compiled.push "cssClass: '#{cssClass}',"  if cssClass?
+    compiled.push "attributes: #{JSON.stringify attributes},"  if attributes?
+    compiled.push "subviews: [#{children.map(@compileNode.bind this).join ','}]"
```
Comment (usirin) [inline src/compiler.coffee:42]: probably need to changing `@compileNode` part to `Compiler.compileNode` would be better.

Comment (szkl) [inline src/compiler.coffee:42]: You don't need to bind that function. It is context-free. Writing it as `Compiler.compileNode` might be better for readability.

Comment (usirin) [inline src/compiler.coffee:42]: Exactly. BTW, the bind part was necessary because of the recursive calls.

**Comment by usirin** [comment]
I am merging this in, will fix those issues in another PR.

**Pull request merged**